src/util/psbt/map/global.rs

```diff
@@ -155,6 +165,55 @@ impl Decodable for Global {
                                 return Err(Error::InvalidKey(pair.key).into())
                             }
                         }
+                        0x01 => {
```

Comment:
Might be nice to have a comment about which PSBT field these hex values represent

Reply:
Also, I think you should change `0u8` to `0x00` on line 140 above